motivaa-backend/motivaa-app/src/test/java/backendtesting/TestUtils.java

```diff
@@ -0,0 +1,108 @@
+package backendtesting;
+
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import org.testng.Assert;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TestUtils {
+
+    public static final String LOCKVERSIONS = "lockVersions";
+    public static final String UPDATEMAP = "updateMap";
+
+
+    public static void updateProcessObjectAndVerifyOutputValues(String environmentHost,
```

Comment:
@cynran  do you think I should move this out to a separate class? Not sure if I am not overusing TestUtils.

Reply:
If you think you will use this in multiple tests, then it is fine here. You can can multiple TestUtil classes if you want to separate them, but then the separation should be clear.